integration_tests/test_domestic_migration.py

```diff
@@ -75,6 +73,18 @@ def test_individuals_move_into_group_quarters(simulants_on_adjacent_timesteps):
         )
 
 
+def get_households_with_reference_person(after, before, movers):
```

Comment:
I find the name of this method a bit confusing -- maybe `get_households_with_stable_reference_person` or something?

Reply:
Wait, I missed this on first read, but this `movers` arg is not used.

Reply:
I'll rename these in my next pr as it'll be easier to manage the git history